maintainers/team-list.nix

```diff
@@ -149,6 +149,19 @@ with lib.maintainers; {
     enableFeatureFreezePing = true;
   };
 
+  cuda = {
+    members = [
+      cigrainger
+      connorbaker
+      knedlsepp
+      samuela
+      SomeoneSerge
```

Comment:
Folks, if you'd rather not get personally `@`-ed by the CI, I guess you might want to remove yourself from this list

Reply:
Please do not add anyone without their explicit permission.

Reply:
Better: let them be added in further PR if they miss the first PR.

Reply:
Fair, let's have explicit PRs